src/wombats/handlers/game.clj

```diff
@@ -157,6 +165,18 @@
                         game-body-params]
            :responses {:200 {:description "add-game response"}}}}})
 
+(defn- set-game-defaults
+  [{:keys [:game/password
+           :game/num-rounds
+           :game/round-intermission
+           :game/start-time] :as game}]
+  (merge game
+         {:game/password (or password "")
```

Comment:
Diatomic restriction. We could just `dissoc` that field. Will make the adjustment in #217.